### ISSUE TITLE
In the iPhone version imei different from the device  

### ISSUE THREAD

**Issue by s2yed** [kevincaicedo/imei_plugin#20]
In the iPhone version
imei different from the device returns random numbers
02E81A16-BAC6-4F02-A825-30B2937AAE82

**Comment by kevincaicedo** [comment]
yes is right, because in iphone is no posible access to the original imei

**Comment by s2yed** [comment]
Is this number similar? In another device, and does each device have a
different number? ٫, Can you deduce it inidfity?

**Comment by kevincaicedo** [comment]
yes in each device is different

**Comment by s2yed** [comment]
Ok thanks
